src/services/consumer.js

```diff
@@ -373,7 +373,7 @@ class Consumer {
                 let conditions = weather[i];
                 try {
                     const weather = Weather.fromClientWeather(conditions.cell.toString(), conditions.data, ts);
-                    await weather.triggerWebhook();
+                    weather.triggerWebhook();
```

Comment:
```suggestion
                    await weather.triggerWebhook();
```
We need await for promises